lib/flat_search/flats/flat.ex

```diff
@@ -0,0 +1,54 @@
+defmodule FlatSearch.Flats.Flat do
+  @moduledoc """
+  Flat module. Schema and changesets
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "flats" do
+    field :unique_id, :string
+    field :title, :string
+    field :link, :string
+    field :price, :integer
+    field :additional_price, :integer
+    field :negotiation, :boolean, default: false
+    field :surface, :integer
+    # field :description, {:array, :string}
+    field :favourite, :boolean
+    field :state, :string
+    field :photo_links, {:array, :string}
+
+    timestamps()
+
+    def changeset(flat, attrs) do
```

Comment:
shouldn't there be `end` after `timestamps()`?

Reply:
Lol, how did it work before